src/aat/resources/features/F-106 - Update Supplementary Data/F-106_Update_Supplementary_Data_Base_C2_PRIVATE.td.json

```diff
@@ -0,0 +1,27 @@
+{
+	"_guid_": "F-106_Update_Supplementary_Data_Base_C2_PRIVATE",
```

Comment:
This file could be removed and all the extra information there could be  added to [F-106_Update_Supplementary_Data_C2_PRIVATE.td.json](https://github.com/hmcts/ccd-data-store-api/pull/2178/files#diff-7f9c631fe3504b0280e16ae6048234ae360b9b5614a82af2693fd652666710c5)